### ISSUE TITLE
NEI crash when pressing U

### ISSUE THREAD

**Issue by 3036Macloud** [ReikaKalseki/Reika_Mods_Issues#1267]
Client crashes when using U in NEI. In single player world it doesn't crash but takes around 1 minute thinking about it but after that is fine. Once in multiplayer world after performing that function in single player its ok.
Paste bin-http://pastebin.com/dVFAdtge

**Comment by ReikaKalseki** [comment]
Try removing mods.

**Comment by 3036Macloud** [comment]
I removed all mods expect NEI and it works fine. Could this be related to forestry which in other post i see has been an issue. Not saying this is the exact same fault but considering it mentions fermenter in the crash log ?
Anyway when I get a chance I'll remove Forestry to check.

**Comment by 3036Macloud** [comment]
So I removed Forestry and still the same so it wasn't that Mod. I then proceeded to remove every Mod in groups. I got down to almost no other mod except all your mods listed and the error disappeared.  The last group of mods was then  re added along with every other mod because the error never returned.
No configs were changed because they where the original ones used.
Is it because your mod require to be installed first to log all recipes ? I'm just guessing but just interested to know for future reference.
